src/button-dropdown/internal.tsx

```diff
@@ -258,7 +258,6 @@ const InternalButtonDropdown = React.forwardRef(
               styles['trigger-item'],
               styles['split-trigger'],
               styles[`variant-${variant}`],
-              mainActionProps.disabled && styles.disabled,
```

Comment:
`disabled` class was removed from styles as a part of #2529, cleaning up jsx too